<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerWindow</class>
 <widget class="QWidget" name="ServerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="startStopButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Server</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="logEditor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>371</width>
     <height>171</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="ServerConsole" name="serverConsole">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>371</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ServerConsole</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">serverconsole.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
